src/System.Net.Security/ref/4.0.0/System.Net.Security.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@weshaggard Is that a new procedure?  Because historically we always create the ref/"old contract" folders.  See System.Net.Http library.